src/lib/utils/options/sources/typedoc.ts

```diff
@@ -243,4 +244,9 @@ export function addTypeDocOptions(options: Pick<Options, "addDeclaration">) {
             "Specify the options passed to Marked, the Markdown parser used by TypeDoc",
         type: ParameterType.Mixed,
     });
+    options.addDeclaration({
+        name: "highlightTheme",
+        help: "Specifies the code highlighting theme.",
+        type: ParameterType.String,
+    });
```

Comment:
Can you add a validation function to this to ensure that the user provided is valid? I believe Shiki exports an array of themes.